src/rating-calculator/components/ChartRecordDataRow.tsx

```diff
@@ -69,6 +69,18 @@ export const ChartRecordDataRow = React.memo((props: Props) => {
                 <div key={idx}>+{r.minRt.toFixed(0)}</div>
               ))
             : '';
+        case ColumnType.NEW_RATING:
+          return record.rating
+            //Used in calculating rating on played charts
+            ? Array.from(record.nextRanks.values()).map((r, idx) => (
+                <div key={idx}>{(Math.floor(record.rating)+r.minRt).toFixed(0)}</div>
```

Comment:
I just realized this formula (= `record.rating + r.minRt`) is incorrect.

When the record is in the rating subjects (top 15 or top 35), minRt is the delta between `r.rating` and rating at next rank.

When the record is outside the rating subject, minRt is the delta between **the lowest rating in the rating subjects** and the rating at next rank.

For example, if my top 15 ends with rating 300 and I have a chart record of rating 290. The next rank for the chart record is 301. In this situation, r.minRt will be `+1` while the new rating should be 301 instead of `290 + 1`.

My suggestion is to ignore `record.rating` and always calculate the rating based on `r.rank`

Reply:
Implemented as suggested. Both `Show played charts` and `Show not yet played charts` can use the same approach